src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -42,6 +42,15 @@ class RsHighlightingAnnotator : AnnotatorBase() {
         holder.newSilentAnnotation(severity).textAttributes(color.textAttributesKey).create()
     }
 
+    // For elements in a RsMacroExpansionReferenceGroup or RsMacroBindingGroup, return a macro color.
+    // Otherwise, return null
+    private fun macroGroupColor(parent: RsElement, element: PsiElement): RsColor? {
```

Comment:
`element` is unused here

Reply:
Yeah, but maybe not in the future. But I'll remove it and re-add it when needed :)